prisma/schema.prisma

```diff
@@ -288,30 +288,53 @@ model CollectionNft {
 }
 
 model ComicPage {
-  id            Int        @id @default(autoincrement())
-  pageNumber    Int        @default(autoincrement())
-  isPreviewable Boolean    @default(false)
-  image         String
-  comicIssue    ComicIssue @relation(fields: [comicIssueId], references: [id], onDelete: Cascade)
+  id            Int                    @id @default(autoincrement())
+  pageNumber    Int                    @default(autoincrement())
+  isPreviewable Boolean                @default(false)
+  translations  ComicPageTranslation[]
+  comicIssue    ComicIssue             @relation(fields: [comicIssueId], references: [id], onDelete: Cascade)
   comicIssueId  Int
 
   @@unique([pageNumber, comicIssueId])
 }
 
+enum Language {
+  En
+  Fr
```

Comment:
Lets not be cheapskates and use the full language name here. `Language.English`, `Language.French` etc.
We don't know what abbreviations are existing out there and it's more readable in the database if we show the full language name.

There are different abbreviation standards for languages as well so we might mess up there at some point :D
![image](https://github.com/d-reader-organization/d-reader-backend/assets/125173470/0b07d0cc-70e1-40ea-8b11-587ee1fe798d)